src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -250,8 +249,6 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
             {ov::element::u32,     ov::element::i32},
             {ov::element::f64,     ov::element::f32},
             {ov::element::boolean, ov::element::u8},
-            {ov::element::i4,      ov::element::i8},
-            {ov::element::u4,      ov::element::u8}
```

Comment:
It's not clear for me what will happen if u4/i4 ops, which are not compressed weights, are not converted into u8/i8. Will plugin handle it somehow or just throw an exception?

Reply:
Applied your patch: https://github.com/openvinotoolkit/openvino/pull/20607/commits/7669c9e5dd0d1329b4ec9dd279ef3acff0faa223